web/src/app/admin/embeddings/modals/ProviderCreationModal.tsx

```diff
@@ -205,8 +220,8 @@ export function ProviderCreationModal({
                 ) : (
                   <TextFormField
                     name="api_key"
-                    label="API Key"
-                    placeholder="API Key"
+                    label={`API Key ${isProxy && "(for non-local deployments)"}`}
+                    placeholder={`API Key `}
```

Comment:
Should there be a space on the end here? Are the backticks useful here?

Reply:
Good catch!